ConnectionsPanel.js

```diff
@@ -420,4 +382,110 @@ ToolsButton.propTypes = {
   count:         PropTypes.number,
 };
 
+const ConnectionsPanelSection = ({ title, children }) => {
+  const { themeStr, interfaceLanguage } = useContext(GlobalStateContext);
+  const theme = getTheme(themeStr);
+  return (
+    <View style={styles.connectionPanelSection}  flexdir={interfaceLanguage === "english" ? null : styles.rtlRow}>
+      {title &&
+        <View style={styles.connectionPanelTitle}>
+          <Text style={[interfaceLanguage === "hebrew" ? styles.heInt : styles.enInt, theme.tertiaryText]}>
+            {title}
+          </Text>
+        </View>
+      }
+      <View>
+        {children}
+      </View>
+    </View>
+  );
+};
+ConnectionsPanelSection.propTypes = {
+  title: PropTypes.string,
+  children: PropTypes.node.isRequired,
+};
+
+const TopButtons = ({relatedHasError, reloadRelated, themeStr, setConnectionsMode, sheet, versionsCount}) => {
+  return (
+    <ConnectionsPanelSection>
+      {!relatedHasError ? null : <ToolsButton
```

Comment:
switch the order of this if statement so it is positive (not using `!`). This tends to be simpler to understand.